packages/babel-plugin-transform-class-properties/src/index.js

```diff
@@ -52,8 +53,8 @@ export default function({ types: t }) {
     inherits: syntaxClassProperties,
 
     visitor: {
-      Class(path, state) {
-        const buildClassProperty = state.opts.loose
+      Class(path) {
+        const buildClassProperty = loose
```

Comment:
This whole assignment can get moved to the top since it only cares about `loose`.